packages/components/src/components/Icon/Icon.tsx

```diff
@@ -179,7 +179,11 @@ export type IconProps = nameOrSvg &
   }
 
 /**
- * A common component to display assets (SVGs).
+ * Convenience component to display Icons that are in .svg file format. A set
+ * of pre-loaded icons can be found in the [mobile-assets](https://www.npmjs.com/package/\@department-of-veterans-affairs/mobile-assets)
+ * package. Custom SVGs can also be used if your project is configured to
+ * import them. See [\@svgr/webpack](https://www.npmjs.com/package/\@svgr/webpack)
```

Comment:
Is the "\" supposed to be there? Noticing it doesn't appear in the name displayed in NPM.

Reply:
Yup. `@` characters need to be escaped with a slash otherwise eslint/tsdoc complains.